proto/viam/service/motion/v1/motion.proto

```diff
@@ -207,8 +207,8 @@ message StopPlanRequest {
   // The name of the motion service
   string name = 1;
 
-  // The root component of the currently executing plan to stop
-  common.v1.ResourceName root_component = 2;
+  // The component of the currently executing plan to stop
+  common.v1.ResourceName component_name = 2;
```

Comment:
@raybjork & @tessavitabile I'd like to also add this change in this PR. Calling this field `root_component` in the `StopPlanRequest` was a typo from a previous iteration of the scope doc. Nowhere else uses the term `root_component` & eliot specifically asked us to use `component_name` everywhere we send a component name.

Can we please roll this fix into this PR?

Reply:
Thats good with me

Reply:
same